staging/src/kubevirt.io/client-go/kubecli/kubevirt.go

```diff
@@ -314,13 +314,7 @@ type VirtualMachineInstanceMigrationInterface interface {
 }
 
 type KubeVirtInterface interface {
-	Get(ctx context.Context, name string, options metav1.GetOptions) (*v1.KubeVirt, error)
-	List(ctx context.Context, opts metav1.ListOptions) (*v1.KubeVirtList, error)
-	Create(ctx context.Context, instance *v1.KubeVirt, opts metav1.CreateOptions) (*v1.KubeVirt, error)
-	Update(ctx context.Context, kubeVirt *v1.KubeVirt, opts metav1.UpdateOptions) (*v1.KubeVirt, error)
-	Delete(ctx context.Context, name string, options metav1.DeleteOptions) error
-	Patch(ctx context.Context, name string, pt types.PatchType, data []byte, patchOptions metav1.PatchOptions, subresources ...string) (result *v1.KubeVirt, err error)
-	UpdateStatus(ctx context.Context, kubeVirt *v1.KubeVirt, opts metav1.UpdateOptions) (*v1.KubeVirt, error)
+	kvcorev1.KubeVirtInterface
 	PatchStatus(ctx context.Context, name string, pt types.PatchType, data []byte, patchOptions metav1.PatchOptions) (result *v1.KubeVirt, err error)
```

Comment:
What is the reason to keep `PatchStatus`?
Do we still need `KubeVirtInterface`?

Reply:
PatchStatus is used in `patchStatusUnstructured` method of the `updater` struct https://github.com/kubevirt/kubevirt/blob/b9e89e2982605f535166065ddb1e8d4a77b4c204/pkg/util/status/status.go#L148-L163
For this reason our `KubeVirtInterface` is still needed, otherwise, we will not have the `PatchStatus` method.
Also, generally, before dropping our interfaces we first need to adjust all the unit tests to use the fakeclient.
This is because when we drop our interface we lose also the mockClient (that is required at the moment for the uni tests) :)

Reply:
Thank you for the explanation.
Could you please add it to the last commit message?

Reply:
Thank you! Done!